app/helpers/application_helper.rb

```diff
@@ -1,6 +1,6 @@
 module ApplicationHelper
   def full_title page_title = ""
-base_title = "Ruby on Rails Tutorial Sample App"
-page_title.empty? ? base_title : "#{page_title} | #{base_title}"
-end
+    base_title = t "ruby_on_rails_tutorial_sample_app"
```

Comment:
trong file helper thì phải khai báo rõ I18n.t
```suggestion
    base_title = I18n.t "ruby_on_rails_tutorial_sample_app"
```